### ISSUE TITLE
Migrate back to Brief

### ISSUE THREAD

**Issue by tanriol** [tanriol/digest#80]
As successful cooperation was established over Brief, I am going to stop maintaining Digest after Brief catches up. Thus, the remaining unmerged fixes should be merged into Brief.
After the Brief version containing all the fixes goes through review and becomes the stable version, a new Digest version will be released with the suggestion to migrate to Brief.

If anyone has reasons not to go back, I'd be interested in hearing the reasons.

Brief's upstream is now https://github.com/ancestorak/brief

**Comment by hurda** [comment]
Great to hear that you're joining forces to move this addon forward.

**Comment by hurda** [comment]
Is it safe to switch to Brief now? Safe in the sense of the database and the state of feeds (unread items etc.) being accepted by the latest Brief-version.

**Comment by tanriol** [comment]
Yes, the current stable version is OK. One or two minor bugs are still there, but defenitely no data loss.

**Comment by hurda** [comment]
Thanks.

**Comment by jnachtigall** [comment]
I think you should leave a note on the github summary page and also on mozilla's addon page about it. I just installed brief, only to learn 2 days later that it is goning to be discontinued :-/
